cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -32,10 +32,7 @@ clean: ## delete generated byte code and coverage reports
 	rm -rf assets
 	rm -rf pii_report
 
-clean_pycrypto: ## temporary (?) hack to deal with the pycrypto dep that's installed via setup-tools
-	ls -d /usr/lib/python3/dist-packages/* | grep 'pycrypto\|pygobject\|pyxdg' | xargs rm -f
```

Comment:
@iloveagent57 You added in this in https://github.com/openedx/edx-cookiecutters/pull/57 but I'm not clear on what problem you were trying to solve. Do you remember?

Here's what happens for me on my Ubuntu laptop:

```
rm: cannot remove '/usr/lib/python3/dist-packages/pycryptodomex-3.11.0.egg-info': Is a directory
rm: cannot remove '/usr/lib/python3/dist-packages/pyxdg-0.27.egg-info': Is a directory
```

I wonder if maybe you weren't in a virtualenv when you were testing?